tests/components/miele/test_init.py

```diff
@@ -195,10 +195,10 @@ async def test_setup_all_platforms(
     assert hass.states.get("switch.washing_machine_power").state == "off"
 
     # Add two devices and let the clock tick for 130 seconds
-    freezer.tick(timedelta(seconds=130))
-    mock_miele_client.get_devices.return_value = load_json_object_fixture(
-        "5_devices.json", DOMAIN
+    mock_miele_client.get_devices.return_value = await async_load_json_object_fixture(
+        hass, "5_devices.json", DOMAIN
     )
+    freezer.tick(timedelta(seconds=130))
```

Comment:
`freezer.tick` needed to be called after setting the `return_value`, as the new `await` could cause a race condition